docs/7.Scan Examples/Git History.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+published: false
+title: SCA scanning
+nav_order: 20
+---
+
+# Git History
+Checkov supports scanning for secrets in git history, to identify and flag secrets which might not be in the current brunches, but are still visible by accessing the git history of the repo.
+Checkov fetches all the available commits, and uses the same strategy as in 'secrets' checkov framework to search for secrets in the diff available from git history.
+Each secret found, can be traced by the first commit in which it appeared, and the last commit, if it has been removed. 
+
+
+## Git History scanning
+
+Git history scan is the same as secrets scan.
+The only difference is that the root directory is the path to either the root git directory or the bare git repo.
+
+```bash
+checkov -d <git dir> --framework secrets --bc-api-key <your_api_key>
```

Comment:
```suggestion
checkov -d <git dir> --scan-secrets-history --bc-api-key <your_api_key>
```
also add an example with `--secrets-history-timeout` or an explanation on when to use this flag.